basetest.pm

```diff
@@ -705,20 +705,24 @@ sub parse_serial_output_qemu {
             my $type    = $regexp_table->{type};
 
             # Input parameters validation
-            die "Wrong type defined for serial failure. Only 'soft' or 'hard' allowed. Got: $type" if $type !~ /^soft|hard|fatal$/;
+            die "Wrong type defined for serial failure. Only 'soft' or 'hard' allowed. Got: $type" if $type !~ /^info|soft|hard|fatal$/;
```

Comment:
I know that the "Only …" part already misses `fatal` but it would still make sense to update it.

Reply:
Nice catch. Changed